tests/integration/test_market.py

```diff
@@ -79,6 +79,21 @@ def test_match_order(bootstrapped_market: Market, stubbed_payer: Account):
     assert sum(1 for _ in asks) == 0
 
 
+@pytest.mark.integration
+def test_settle_fund(
+    bootstrapped_market: Market,
+    stubbed_payer: Account,
+    stubbed_quote_wallet: Account,
+    stubbed_base_wallet: Account,
+):
+    open_order_accounts = bootstrapped_market.find_open_orders_accounts_for_owner(stubbed_payer.public_key())
+
+    for open_order_account in open_order_accounts:
+        bootstrapped_market.settle_funds(
+            stubbed_payer, open_order_account, stubbed_base_wallet.public_key(), stubbed_quote_wallet.public_key()
```

Comment:
```suggestion
            stubbed_payer, open_order_account, stubbed_base_wallet.public_key(), stubbed_quote_wallet.public_key(), opts=TxOpts(skip_confirmation=False)
```

Okay for this test, let's just make sure there are no errors in the responses.